components/brave_rewards/browser/rewards_service_factory.cc

```diff
@@ -117,7 +117,7 @@ void RewardsServiceFactory::Observe(
     int type,
     const content::NotificationSource& source,
     const content::NotificationDetails& details) {
-#if BUILDFLAG(BRAVE_REWARDS_ENABLED)
+#if BUILDFLAG(BRAVE_REWARDS_ENABLED) && !defined(OS_ANDROID)
```

Comment:
why this change?

Reply:
not exist on Android

Reply:
this define is related to defines above, so I guess it should be reverted or left depending on what we do with defines above.

Reply:
Removed